File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p04012/p04012_num2_parsed.py
w = input()

# Count the occurrence of each lowercase letter
count = [0] * 26
for c in w:
    index = ord(c) - ord('a')
    count[index] += 1

# Check if each lowercase letter occurs even number of times
for i in range(26):
    if count[i] % 2 != 0:
        print("No")
        exit()

print("Yes")

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p04012/p04012_num2_parsed.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    string w;
    cin >> w;

    vector<int> count(26, 0);

    // count the occurences of each letter
    for (char letter : w) {
        count[letter - 'a']++;
    }

    // check if all counts are even
    bool beautiful = true;
    for (int i = 0; i < 26; i++) {
        if (count[i] % 2 != 0) {
            beautiful = false;
            break;
        }
    }

    // print the result
    if (beautiful) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }

    return 0;
}